packages/core/src/storage/migration/UpdateAssistant.ts

```diff
@@ -192,7 +192,7 @@ export class UpdateAssistant<Agent extends BaseAgent<any> = BaseAgent> {
   }
 
   private getBackupPath(backupIdentifier: string) {
-    return `${this.fileSystem.basePath}/afj/migration/backup/${backupIdentifier}`
+    return `${this.fileSystem.tempPath}/migration/backup/${backupIdentifier}`
```

Comment:
Do we want to use the temp path for this? If you start a migration and close the app in the process, it may be nice that we could recover it and thus have to manually delete the backup (this is done I think already afterwards)

Reply:
Yes, that's not a very good idea as it is unspecified when the OS will clean up the files, especially large ones.

But if we are going to use data directory I think we should automatically delete the file once we successfully finish the migration or restore the backup from it. This is something not happening right now (at least on framework level), so I can add it (and some test for it) within this PR.

Reply:
Just a remark regarding something I found during this update: framework produces a backup-error file when restoring a wallet. I guess the value of doing so is to be able to do some further debugging. Anyway, in this case this file will remain in the file system.

It's not supposed to be a common case, so it's fine for me but just would like to note this if you see it could represent an issue.